code/game/gamemodes/steal_items.dm

```diff
@@ -47,9 +47,9 @@
 	protected_jobs = list("Captain")
 	location_override = "the Captain's Office"
 
-/datum/theft_objective/captains_rapier
-	name = "the captain's rapier"
-	typepath = /obj/item/melee/rapier
+/datum/theft_objective/captains_sabre
+	name = "the captain's sabre"
```

Comment:
It's not curved, so it would actually be more accurately described as a falchion.